src/libraries/Common/tests/System/Net/Http/Http2LoopbackConnection.cs

```diff
@@ -100,6 +100,8 @@ private async Task ReadPrefixAsync()
                 // The contents of what we send don't really matter, as long as it is interpreted by SocketsHttpHandler as an invalid response.
                 await _connectionStream.WriteAsync(Encoding.ASCII.GetBytes("HTTP/2.0 400 Bad Request\r\n\r\n"));
                 _connectionSocket.Shutdown(SocketShutdown.Send);
+
+                throw new Exception("HTTP/1.1 request sent to HTTP/2 connection.");
```

Comment:
A comment might be useful here:

```suggestion
                // If WinHTTP doesn't support streaming a request without a length then it will fallback
                // to HTTP/1.1. Throwing an exception to detect this case in WinHttpHandler tests. 
                throw new Exception("HTTP/1.1 request sent to HTTP/2 connection.");
```